_includes/manuals/1.16/4.1.3_roles_and_permissions.md

```diff
@@ -18,7 +18,7 @@ The seeded Organization admin role is similar to the Manager role. They are both
 
 #### Filters
 
-Filters are defined within the context of a role, clicking on the 'filters and permissions' link. A filter allows an user to choose a **resource** (Hosts, Host groups, etc...) and the **permissions** that should be granted for that resource. After a filter has been created, users given a role containing this filter will have the permissions for the resource specified at the filter.
+Filters are defined within the context of a role, clicking on the 'filters and permissions' link. A filter allows a user to choose a **resource** (Hosts, Host groups, etc...) and the **permissions** that should be granted for that resource. After a filter has been created, users given a role containing this filter will have the permissions for the resource specified at the filter.
```

Comment:
The first sentence comma is not needed and I'd consider replacing it with `by` so that you get `context of a role by clicking on the..`